webapp/src/game/game.service.ts

```diff
@@ -102,4 +102,21 @@ export class GameService {
       player.id === game.player_black.id || player.id === game.player_white.id
     );
   }
+
+  checkPlayerTurn(game: Game, player: User) {
```

Comment:
Not a big fan of `check` here, maybe `isPlayerTurn` is better. `check` refers to assertions imho

Reply:
I share your opinion.  But I wrote an another function called `isGameBelongToPlayer` and I was asked to change it to `checkGameBelongToPlayer`. So I used the same rules for this function